_includes/nav-docs.html

```diff
@@ -1,21 +1,18 @@
-{% for section in site.data.nav_docs %}
-  <h3>{{ section.title }}</h3>
-  <ul class="sidebar-links">
-    {% for item in section.items %}
-      <li>
-        <a href="{{ site.baseurl }}/docs/{{ section.id }}/{{ item.id }}.html"
-          {% if page.id == item.id %} class="active"{% endif %}>{{ item.title }}</a>
-        {% if item.subitems %}
-          <ul>
-            {% for subitem in item.subitems %}
-              <li>
-                <a href="{{ site.baseurl }}/docs/{{ section.id }}/{{ subitem.id }}.html"
-                  {% if page.id == subitem.id %} class="active"{% endif %}>{{ subitem.title }}</a>
-              </li>
-            {% endfor %}
-          </ul>
-        {% endif %}
-      </li>
-    {% endfor %}
-  </ul>
-{% endfor %}
+    {% assign pages = site.pages | sort:"weight"  %}
+    {% assign current_array = page.url | remove_first: "/" | split:"/" %}
+    {% assign current_section = current_array[0] %}
+        <h3>{{ current_section | replace: '-',' ' | capitalize }}</h3>
```

Comment:
extra indentation here. also, please check your https://github.com/symphonycms/docs.getsymphony.com/blob/gh-pages/.editorconfig (this is not mandatory, but good enforce coding standards, which is 2 spaces for indentation)

Thanks!